test/client.e2e.ts

```diff
@@ -2,8 +2,14 @@ import { Offset, ProximaStreamClient, SingleStreamDbRegistry, StreamRegistryClie
 import { strict as assert } from "assert";
 import { firstValueFrom, take, toArray } from "rxjs";
 
-const testStream = "proxima.eth-main.blocks.1_0"
-const streamDbUri = "streams-mirror.buh.apps.proxima.one:443";
+const streamTests = [{
+  stream: "proxima.eth-main.blocks.1_0",
+  timestamp: 1438473608000,
+}, {
+  stream: "proxima.exchange-rates.0_1",
+  timestamp: 1438473600000,
+}];
+const streamDbUri = "0.0.0.0:50051";
```

Comment:
This won't work in the CI. It's important that these tests check the status of our buh cluster